homework-2/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace PersonalInformation
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            Person person1 = new Person();
+            Person person2 = new Person("Brahm", 21);

```

Comment:
Whenever there is a scenario where you have to create Obj1, Obj2 and so one.. make sure you don't use Numeric numbers.. you can name these variables as `PersonOne` and `Person2`.